### DISCUSSION TITLE
Version 1.2.5 publication

### DISCUSSION THREAD

**Discussion by erenard** [privateOmega/html-to-docx#109] [Q&A] [answered]
Hello,

Just a reminder to publish the last version of `html-to-docx` on npmjs.
I'd rather use it from npm than from a local fork !

Thanks

**Comment by privateOmega** [comment]
@erenard Sure, I will publish a new version by tomorrow.

**Comment by privateOmega** [comment, marked as answer]
Released version 1.3.1. Hope it works for you.
